src/app/components/LeftMenu/PartyCardList/Card/index.tsx

```diff
@@ -27,6 +27,7 @@ const Box = styled.div`
 `;
 
 const LeftWrapper = styled.div`
+    width:350px;
```

Comment:
this should be worked fine

I wonder the reason why changed this line

Reply:
Without it, width of the right wrapper (red one) changed for every card.